website/project/model.py

```diff
@@ -1091,8 +1096,11 @@ def is_registration_of(self, other):
     @property
     def forks(self):
         """List of forks of this node"""
-        return list(self.node__forked.find(Q('is_deleted', 'eq', False) &
-                                           Q('is_registration', 'ne', True)))
+        return Node.find(
+            Q('is_deleted', 'eq', False) &
+            Q('is_registration', 'ne', True) &
+            Q('forked_from', 'eq', self._id)
```

Comment:
I would have expected this to have been handled already. Was this missing from a backref removal PR or is there an open PR that's going to have a merge conflict?